newspack-theme/inc/logo-resizer.php

```diff
@@ -135,10 +136,9 @@ function newspack_customize_logo_resize( $html ) {
 			}
 		}
 
-		@media (min-width: 1200px) {
-			.h-stk:not(.h-sub) .site-header .custom-logo {
-				max-width: ' . $sticky['width'] . 'px;
-				max-height: ' . $sticky['height'] . 'px;
+		@media (max-width: 1199px) and (min-width: 782px) {
+			.h-stk.h-cl:not(.h-sub) .site-header .custom-logo {
+				max-width: 100%;
```

Comment:
Works as described for most cases. I'm still getting stretched-out logos between `782` and `1199` viewports for small and x-small versions.

![image](https://user-images.githubusercontent.com/820752/147945979-7e8f8070-1845-4cdb-900f-5f451e5e3240.png)

 Adding `width: auto;` seems to fix the issue, since `height` is the fixed value.

```suggestion
				max-width: 100%;
				width: auto;
```

Reply:
Thanks for testing this, @miguelpeixe (and apologies for the very late reply!) 

I can see that issue, too 🤦‍♀️ I'm not sure how I missed it, I think I must have made a last second change when working on this! 

I've added the `width: auto` and did some testing; it seemed to fix the issue for me, too, and didn't seem to introduce any others! Please let me know if you see any other issues, though!